docs/toc.yml

```diff
@@ -286,6 +289,8 @@ items:
           href: develop/get-javascript-intellisense-in-visual-studio.md
         - name: Convert JavaScript projects to TypeScript
           href: develop/convert-javascript-to-typescript.md
+      - name: Copy Script Lab snippet to Yo Office
+        href: overview/create-an-office-add-in-from-script-lab.md
```

Comment:
Consider adding terms to `displayName` to help with discoverability during TOC filtering especially among devs new to Office Add-ins. Some examples I can think of are "code" or "code sample".

Reply:
Good suggestion. I settled on 'code samples' because the autocomplete will work if someone types 'code sample'.

Can you add some tags to the appropriate Outlook articles? Not a rush, but I feel like you know which ones are the correct choices to tag.